src/util/SoundUtil.ts

```diff
@@ -17,7 +17,7 @@ const getSoundWithExtension = (sound: string) => {
 };
 
 export const getSoundsWithExtension = () => getSoundsFromSoundFolder().map(getSoundWithExtension);
-export const getSounds = () => getSoundsWithExtension().map(sound => sound.name);
+export const getSounds = () => getSoundsWithExtension().map(sound => sound.name.toLowerCase());
```

Comment:
I think sounds are already only accepted in all lowercase, as per https://github.com/markokajzer/discord-soundbot/blob/master/src/commands/manage/add/validator/BaseValidator.ts#L9

Reply:
I did this on my instance to ensure that no matter the capitalization on the mp3 file name it would import into the map as lowercase.  Then in the other change I added the toLowerCase to ensure that all incoming commands sent to the bot were also in lower case.  One of these might be redundant as I am very amateur at coding.  I just figured I would share my changes.

Reply:
I noticed that when the bot received a command that is was case sensitive these changes just ensure that no matter the case of the incoming message it will be interpreted as lower case.  This might not be a change you want to implement fully as some use cases might desire case sensitivity but I prefer it not to be.